app/javascript/components/game_map.js

```diff
@@ -0,0 +1,63 @@
+import { Axis } from './axis'
+
+class GameMap {
+  constructor(selector) {
+    this._selector = selector
+  }
+
+  addMap() {
+    // Add component html to DOM
+    this._element.innerHTML = this._html()
```

Comment:
Should this check to make sure the element exists, and abort the process if it doesn't?

That would prevent the need for checking [here](app/javascript/components/axis.js)